backend/core/src/migration/1691377869351-new-status-fields.ts

```diff
@@ -0,0 +1,35 @@
+import { MigrationInterface, QueryRunner } from "typeorm"
+
+export class newStatusFields1691377869351 implements MigrationInterface {
+  name = "newStatusFields1691377869351"
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
```

Comment:
I don't ✨ think ✨ this results in status data loss but would love a second pair of eyes

Reply:
I think that this would work, you may want to look into the solution: https://stackoverflow.com/questions/1771543/adding-a-new-value-to-an-existing-enum-type/7834949#7834949

seems like a cleaner solution with less possibility of altering existing data

Reply:
I'd also push for the way in the stackoverflow link

Reply:
Updated! As far as I can tell from that resource looks like there isn't a better alternative for the down migration.